bin/yb-ctl

```diff
@@ -616,8 +632,24 @@ class ClusterControl:
             json.dump(self.cluster_config, config_file)
 
     def is_postgres_enabled(self):
-        return (self.cluster_config.get("enable_postgres") or
-                self.cluster_config.get("enable_ysql") or self.options.enable_ysql)
+        return self.cluster_config.get(
+            "enable_postgres",
+            self.cluster_config.get(
+                "enable_ysql", self.options.enable_ysql))
```

Comment:
I think the logic here is that, if we have a line in the config, we'll use it (so any new deployments, AFTER the create), otherwise (old deployments or just creating), we use the arg value.